app/router.js

```diff
@@ -45,63 +77,65 @@ export async function router(auth, id, payload, verbose) {
     console.log(
       `${id} Closing issue ${payload.issue.html_url}, reason: ${reason} ${actorRequest}`
     );
-    await closeIssue(
-      await getAuthToken(auth, payload.installation.id),
-      sourceRepo,
-      payload.issue.node_id,
-      reason
-    );
+    await closeIssue(authToken, sourceRepo, payload.issue.node_id, reason);
     return;
   }
 
-  const reopenMatches = reopenMatcher(payload.comment.body);
+  const reopenMatches = commands.reopen.matches;
   if (reopenMatches) {
     console.log(
       `${id} Re-opening issue ${payload.issue.html_url} ${actorRequest}`
     );
-    await reopenIssue(
-      await getAuthToken(auth, payload.installation.id),
-      sourceRepo,
-      payload.issue.node_id
-    );
+    await reopenIssue(authToken, sourceRepo, payload.issue.node_id);
     return;
   }
 
-  const labelMatches = labelMatcher(payload.comment.body);
+  const labelMatches = commands.label.matches;
   if (labelMatches) {
     const labels = extractCommaSeparated(labelMatches[1]);
+    const result = commands.label.enabled(octokit, config, labels);
```

Comment:
I think you need to separate what is enabled and what is allowed.

don't think an method call `enabled` should lookup whether the user is allowed to perform action.